src/components/AssetDetailsPanel/stories/helper.tsx

```diff
@@ -0,0 +1,38 @@
+import { Asset } from '@cognite/sdk';
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import {
+  ASSET_LIST_CHILD,
+  ASSET_ZERO_DEPTH_ARRAY,
+} from '../../../mocks/assetsListV2';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+
+class CogniteClient extends MockCogniteClient {
```

Comment:
We don't need here such complex implementation here. Better to replace it with something simpler, like:
```typescript
class CogniteClient extends MockCogniteClient {
  assets: any = {
    retrieve: () =>
      new Promise(resolve => {
        setTimeout(() => {
          resolve([fakeAsset]);
        }, 1000);
      }),
  };
}
```